CHANGELOG.md

```diff
@@ -2,13 +2,121 @@
+  qualified paths. Additionally, a new configuration key called `error_mode` has been added to the context statements group.
+  This key determines how the processor reacts to errors that occur while processing that specific group of statements.
+  When provided, it overrides the top-level error mode, offering more granular control over error handling.
+
+- (Contrib) `pkg/stanza`: Allow users to configure initial buffer size ([#37786](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37786))
+- (Contrib) `vcenterreceiver`: Adds three more vCenter virtual machine performance metrics ([#37488](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37488))
+- (Contrib) `k8sclusterreceiver`: Adds new descriptive attributes/metadata to the k8s.namespace and the container entity emitted from k8sclusterreceiver. ([#37580](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37580))
+  - Adds the following attributes to k8s.namespace entity:
+    - k8s.namespace.phase: The phase of a namespace indicates where the namespace is in its lifecycle. E.g. 'active', 'terminating'
+    - k8s.namespace.creation_timestamp: The time when the namespace object was created.
+  - Adds the following attributes to container entity:
+    - container.creation_timestamp: The time when the container was started. Only available if container is either in 'running' or 'terminated' state.
+
+- (Contrib) `splunkenterprisereceiver`: Added a new `splunk.health` metric. ([#36695](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36695))
+- (Contrib) `pkg/ottl`: Introduce ToLowerCase converter function ([#32942](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32942))
+- (Contrib) `pkg/ottl`: Introduce ToSnakeCase converter function ([#32942](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32942))
+- (Contrib) `pkg/ottl`: Introduce ToUpperCase converter function ([#32942](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32942))
+- (Contrib) `resourcedetectionprocessor`: add the Dynatrace detector to the resource detection processor ([#37577](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37577))
+- (Contrib) `googlecloudmonitoringreceiver`: support use [monitoring filters](https://cloud.google.com/monitoring/api/v3/filters) to filter metrics ([#36898](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36898))
+- (Contrib) `pkg/translator/jaeger/internal/jaeger`: Remove jaeger model/converter/thrift/jaeger dependency ([#37820](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37820))
+- (Contrib) `pkg/translator/zipkin/internal/zipkin`: Remove jaeger model/converter/thrift/zipkin dependency ([#37795](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37795))
```

Comment:
These are go api changes, not needed

Reply:
Removed the go api dependencies like this I could identify.